fixieai/agents/code_shot.py

```diff
@@ -30,6 +30,13 @@
 _AGENT_ID_JWT_CLAIM = "aid"
 
 
+@pydantic_dataclasses.dataclass
+class ModelSettings:
+    model: Optional[str] = None
```

Comment:
I think it's good to leave this field as string, as opposed to an SDK enum that gets updated. But I think we should leave a pointer in the docstring to a docs.fixie.ai page that has the list of models that are available